ci/deps/actions-310.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
   - pytest>=7.0.0
   - pytest-cov
   - pytest-xdist>=2.2.0
-  - pytest-asyncio>=0.17
+  - psutil
```

Comment:
`psutil` was recently removed from all of the these files. Could you remove them?